### DISCUSSION TITLE
Seit der neuen Version kann man nix mehr senden von der Box zum Tablet umgekehrt schon

### DISCUSSION THREAD

**Discussion by KittyBua** [localsend/localsend#1897] [L-Deutsch]
eit der neuen Version kann man nix mehr senden von der Box zum Tablet umgekehrt schon

Es kommt immer keine Berechtigung bitte in Einstellungen aktiv machen aber bei Einstellungen und dann bei erweiterte Einstellungen kommt nix mehr darunter 

Bitte fixen 

Danke

**Comment by Tienisto** [comment]
Kannst du ein Screenshot erstellen, wo die Fehlermeldung zu erkennen ist?

**Comment by KittyBua** [comment]
![IMG_20241014_151701](https://github.com/user-attachments/assets/67ee3fff-64f6-48a0-acc4-4532debc7982)

Hier also ich kann zum Beispiel von meinen Tablet auf die Box senden das ja und alles 

Aber von der Box zum Tablet zurück nicht weder eine Datei oder Ordner etc 

Und ja Berechtigung ist alles gegeben auch Speicher etc 

Aber funktioniert nicht leider danke

**Comment by KittyBua** [comment]
Es ist Android 11 auf der Box und am Tablet Android 13 

Strong Leap s3 plus Box

Nokia t20 Tablet 

Danke alles andere bzw andere Apps bzw Speicher etc funktioniert alles nur nicht senden von Box zu andere Geräte

**Comment by KittyBua** [comment]
Wenn ich Datei senden will kommt das Bild oben 

Wenn ich einen ganzen Ordner senden will schließt die App 

Vor den letzen Update ging noch alles 

Zu 100% mfg

**Reply by Tienisto** [reply]
App Version 1.13 war lange Zeit die letzte Version, die von Google zugelassen wurde. 1.14 wurde von Google abgelehnt, weil die App bis dahin "vollständigen" Zugriff auf den Speicher hatte. 1.15 wurde die Berechtigung entfernt und stattdessen wird auf die Schnittstelle für "Berechtigung für die vom Nutzer ausgewählten Dateien" zugegriffen. Das funktioniert anscheinend nicht bei allen Gerätetypen. Also du kannst 1.13.1 apk auf deinem Tablet runterladen und von dort aus auf deine Box übertragen und installieren.

Ich schaue, wie man das lösen kann.

**Comment by KittyBua** [comment]
OK danke ja mit der alten geht es natürlich noch 

Ja es geht nicht ums Tablet also von der Box zum Tablet 

Aber ja mit der alten geht es 

Ja ich weiß darum wurden bereits viele Apps entfernt blödes Google 😁 auch mein liebes Cosmi iptv Player 

Aber andere haben es dann gefixt nach dem man es gemeldet hatte wie iptv Proxy , my Play iptv etc die haben das mit externen Explorer gelöst das ganze 

Danke trotzdem 

Wäre toll eine Lösung zum finden \
Zur Not geht natürlich Dateimanager+ oder cx oder ftp APK das geht alles auch über rs und mit den Apps to TV geht es auch noch 

Wäre toll eine Lösung zum finden danke 😃
